http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -786,13 +786,21 @@ class DataprocInterpreter[F[_]: Parallel](
       val sparkMemoryConfigRatio = config.dataprocConfig.sparkMemoryConfigRatio.getOrElse(0.8)
```

Comment:
Is `sparkMemoryConfigRatio` still used? Shall we change it to 0.9 then reuse this config?